core/transport/src/main/proto/grpc/version.proto

```diff
@@ -8,6 +8,9 @@ package alluxio.grpc.version;
 
 
 enum ServiceType {
+  reserved 12;
+  reserved "FILE_SYSTEM_WORKER_WORKER_SERVICE";
```

Comment:
Add a comment to clarify that `BLOCK_WORKER_CLIENT_SERVICE` is now what used to be `FILE_SYSTEM_WORKER_WORKER_SERVICE`

Reply:
Done